content/applications/finance/fiscal_localizations/united_kingdom.rst

```diff
@@ -227,3 +243,184 @@ use our connector as an alternative payroll solution.
+     - `l10n_uk_reports_cis`
+     -  - Allows sending the Monthly return to |HMRC|
+        - CIS Deduction (GB) report for UK construction industry
+   * - :guilabel:`UK - HMRC API`
+     - `l10n_uk_hmrc`
+     -  - Includes the basics of |HMRC|.
+
+.. note::
+   Installing the :guilabel:`UK - Construction Industry Scheme` module installs the other module
+   simultaneously.
+
+To enable the :guilabel:`Test` mode and use test credentials, activate
+:ref:`developer mode <developer-mode>` and go to :menuselection:`Settings --> Technical -->
+System Parameters`. Then search for `l10n_uk_hmrc.api_mode`. Select it and switch the value from
+`production` to `test`.
+
+.. _localization/united-kingdom/cis-monthly-returns:
+
+Monthly returns
+---------------
+
+Monthly returns only work for vendor bills and vendor refunds. To submit a complete return to
+|HMRC|, several steps must be followed to report all payments made to subcontractors under the
+scheme during the previous tax month:
+
+- :ref:`localization/united-kingdom/cis-contractor-setup`
+- :ref:`localization/united-kingdom/cis-subcontractor-setup`
+- :ref:`localization/united-kingdom/cis-vendorbills`
+- :ref:`localization/united-kingdom/cis-monthly-return-sending`
+
+.. _localization/united-kingdom/cis-contractor-setup:
+
+Contractor setup
+~~~~~~~~~~~~~~~~
+
+To configure the company |HMRC| information, go to the :guilabel:`Settings` app. In the
+:guilabel:`Companies` section, click :guilabel:`Manage Companies` and select the company. Open the
+:guilabel:`HMRC` tab and configure the information in the :guilabel:`HMRC Credentials` and the
+:guilabel:`Contractor details` sections. All the fields are mandatory.
+
+.. _localization/united-kingdom/cis-subcontractor-setup:
+
+Subcontractor setup
+~~~~~~~~~~~~~~~~~~~
+
+In the :guilabel:`Contacts` app, open the subcontractor's contact form and select the
+:guilabel:`Accounting` tab. In the :guilabel:`HMRC Details` section, enable the
+:guilabel:`Construction Industry Scheme` option to unlock the additional fields.
+
+By default, the :guilabel:`Deduction rate` is set to 30%. To modify it, a
+:guilabel:`Verification Number` is required. The verification number is provided by |HMRC| when
+verifying the contact. Make sure to check the status of the subcontractor and update the
+:guilabel:`Deduction Rate` accordingly.
+
+The :guilabel:`Forename` and :guilabel:`Surname` fields are mandatory if the contact is set to
+:guilabel:`Individual`.
+
+.. _localization/united-kingdom/cis-vendorbills:
+
+Vendor bills
+~~~~~~~~~~~~
+
+Additional |CIS| tax must be applied to labour line items on vendor bills. Material line items
+on vendor bills do not apply. The tax rate corresponds to the subcontractor's
+:guilabel:`deduction rate`: :guilabel:`O% CIS`, :guilabel:`20% CIS` or :guilabel:`30% CIS`. To apply
+this, in the :guilabel:`Invoice Lines` section of the vendor bill, select the appropriate |CIS| tax
+rate in the :guilabel:`Taxes` column of the labour line items.
```

Comment:
```suggestion
The appropriate |CIS| tax must be applied to **labor items** on vendor bills based on the subcontractor's
:guilabel:`Deduction Rate`: :guilabel:`0% CIS`, :guilabel:`20% CIS` or :guilabel:`30% CIS`. To apply
the rate, go the :guilabel:`Invoice Lines` section of the vendor bill and select the appropriate |CIS| tax
rate in the :guilabel:`Taxes` column of the **labor** items.
```
- deduction rate: because it's a field label
- rephrasing: to try and minimize repeating tax rate
- labor: US spelling
- I added bold to emphasize the fact that this is necessary for labor items but you could also add a list item in the note below to repeat that *The CIS tax rate is not necessary for material items on vendor bills.* if you think it's necessary.

Reply:
@auva-odoo Much better like this, thanks! I thought I should use "labour" since it's a UK page ;) 
It is a great idea to add a list item in the note 👍